README.md

```diff
@@ -47,13 +47,83 @@ This demo contains below applications.
 
 ### Automation of Applications in OpenShift.
 
-TODO: -The below deployment will be achieved using Jenkins Pipelines CICD approach.
+**Build and deploy with pipelines**
 
-     • Deploying application to DEV Environment.
-     • Deploying application from DEV to UAT Environment.
-     • Deploying application in Prod Environment.
-     • Deploying application to Prod DR Environment.
-     • Publishing API’s in 3scale : (By using Operator or ansible playbooks).
+***Install OpenShift Container Platform 3.5 in [CDK 3.0](https://developers.redhat.com/products/cdk/overview/)***
 
+Download the git repository by either forking, or simply cloning it. 
 
+```
+git clone https://github.com/myeung18/IntegrationApp-Automation.git
+```
+Start up your OpenShift environment by running
```

Comment:
Maybe add that if running minishift and not OpenShift, start up your environment by...